### DISCUSSION TITLE
Where is allowDuplicates property in mobile_scanner v3.0.0

### DISCUSSION THREAD

**Discussion by giftty** [juliansteenbakker/mobile_scanner#506] [Q&A]
i can seem to get the scanner detection to delay it scan to quickly when used for qrcodes

**Comment by BBQRIBS72** [comment]
From what I can see, The allowDuplicates property is gone from the MobileScanner but has been replaced by DetectionSpeed.noDuplicates in a MobileScannerController

e.g.
`MobileScanner(
          controller: MobileScannerController(
            detectionSpeed: DetectionSpeed.noDuplicates,
          ),
)`
